packages/react/react.ts

```diff
@@ -36,14 +46,30 @@ export const optimize = (
     return createElement(
       Fragment,
       null,
-      patch.current
-        ? null
-        : createElement('div', { style: { display: 'none' }, ref: marker }),
+      createElement(Marker, { ref: marker }),
       createElement(Effect, { effect }),
     );
   };
 };
 
+const Marker: FunctionComponent<{ ref: RefObject<HTMLDivElement> }> = memo(
+  forwardRef((_, ref: any) => {
+    useInsertionEffect(() => {
```

Comment:
Currently, the implementation first replaces the temporary <div> marker with the actual component, then on unmount it replaces it back in order to respect the React renderer. This is _very_ expensive, is there any way to trick the React renderer to pretend that this node doesn't exist?

Reply:
I'll take a look and try to learn what they're doing. If you have any ideas of how this could be implemented (I think our case should be much easier than signals), feel free to suggest